ynput/OpenPype
# Maya: Extract look is not AYON compatible - OP-5375

## Changelog Description
The textures that would use hardlinking are going through texture processors. Currently all texture processors are hardcoded to copy texture instead of respecting the settings of forcing to copy.
The texture processors were last modified 4 months ago, so effectively all clients that are on any pipeline updated in the last 4 months wont be utilizing hardlinking at all, since the hardcoded texture processors will copy texture no matter the OS.

This opts for completely disabling the hardlinking feature, while we figure out what to do about it.

## Testing notes:
1. Create look in Maya and disable `Force copy`.
2. Publish and verify the textures are being copied instead of hardlinked.
